etcdserver/cluster.go

```diff
@@ -253,6 +253,23 @@ func (c *cluster) ValidateConfigurationChange(cc raftpb.ConfChange) error {
 				return ErrPeerURLexists
 			}
 		}
+
+		nrRunningNodes := 0
```

Comment:
can we move this code block to a func called `checkLiveness` (maybe a better func name)?